Explorer/Assets/DCL/Roads/GPUInstancingPlayground/GPUInstancingServicePlayground.cs

```diff
@@ -0,0 +1,152 @@
+﻿using DCL.Roads.GPUInstancing.Playground;
+using DCL.Roads.Settings;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using UnityEngine.ResourceManagement.AsyncOperations;
+using Utility;
+
+namespace DCL.Roads.GPUInstancing
+{
+    [ExecuteAlways]
+    public class GPUInstancingServicePlayground : MonoBehaviour
+    {
+        public ComputeShader FrustumCullingAndLODGenComputeShader;
+        public ComputeShader IndirectBufferGenerationComputeShader;
+
+#if UNITY_EDITOR
```

Comment:
Aren't playground things editor only already? Anyway, the #if is either not needed, or is a mistake. Surrounding serialized fields with #ifs causes trouble, so please don't.